src/authentication/contentAuth.ts

```diff
@@ -70,21 +70,21 @@ export class ContentAuth extends AlfrescoApiClient {
 
     /**
      * login Alfresco API
-     * @param  username:   // Username to login
-     * @param   password:   // Password to login
+     * @param  username Username to login
+     * @param  password Password to login
      *
      * @returns A promise that returns {new authentication ticket} if resolved and {error} if rejected.
      * */
-    login(username: string, password: string): Promise<any> {
+    login(username: string, password: string): Promise<string> {
         this.authentications.basicAuth.username = username;
         this.authentications.basicAuth.password = password;
 
-        let loginRequest: any = {};
-
-        loginRequest.userId = this.authentications.basicAuth.username;
-        loginRequest.password = this.authentications.basicAuth.password;
+        const loginRequest = new TicketBody({
+            userId: this.authentications.basicAuth.username,
+            password: this.authentications.basicAuth.password
+        });
 
-        let promise: any = new Promise((resolve, reject) => {
+        let promise: any = new Promise<string>((resolve, reject) => {
             this.authApi.createTicket(loginRequest)
                 .then((data: any) => {
```

Comment:
Can we get rid of this `any`? This should return something like 
<img width="468" alt="Zrzut ekranu 2023-08-14 o 07 33 33" src="https://github.com/Alfresco/alfresco-js-api/assets/113341662/2c8b7fa5-dbf5-4ae3-a91c-26ff9befc1c3"> defined here: https://api-explorer.alfresco.com/api-explorer/?urls.primaryName=Authentication%20API#/authentication/createTicket